src/v/pandaproxy/schema_registry/store.h

```diff
@@ -494,9 +494,19 @@ class store {
     }
 
     ///\brief Clear the compatibility level for a subject.
-    result<bool> clear_compatibility(const subject& sub) {
+    result<bool> clear_compatibility(seq_marker marker, const subject& sub) {
         auto sub_it = BOOST_OUTCOME_TRYX(
           get_subject_iter(sub, include_deleted::yes));
+        auto& markers = sub_it->second.written_at;
+        markers.erase(
+          std::remove_if(
+            std::begin(markers),
+            std::end(markers),
+            [marker](auto sm) {
+                return sm.key_type == marker.key_type && sm.seq && marker.seq
+                       && *sm.seq == *marker.seq;
```

Comment:
Clean this up obviously 😇

Reply:
Just added a comment to the `seq_marker::operator==`, but I'm having trouble convincing myself that config records don't have a seq value on the key...I see them added to `config_key` in the write routine and whatnot.

Perhaps I'm missing some context with respect to compat format specifically? Could you elaborate a bit?

Reply:
The `seq` and `node` are particular to Redpanda, other schema registries may not write those to the key. By supporting the compatible format, it's easier to support upgrading from another schema registry (the topic contents can be read directly). In the case where there is no `seq` or `node`, it should be considered that a collision could not have occurred.